tests/subsys/event_manager/src/events/data_event.c

```diff
@@ -13,19 +13,20 @@ static void profile_data_event(struct log_event_buf *buf,
 	struct data_event *event = cast_data_event(eh);
 
 	ARG_UNUSED(event);
-	profiler_log_encode_u32(buf, event->val1);
-	profiler_log_encode_u32(buf, event->val2);
-	profiler_log_encode_u32(buf, event->val3);
-	profiler_log_encode_u32(buf, event->val1u);
-	profiler_log_encode_u32(buf, event->val2u);
-	profiler_log_encode_u32(buf, event->val3u);
+	profiler_log_encode_8(buf, event->val1);
+	profiler_log_encode_16(buf, event->val2);
+	profiler_log_encode_32(buf, event->val3);
+	profiler_log_encode_8(buf, event->val1u);
+	profiler_log_encode_16(buf, event->val2u);
+	profiler_log_encode_32(buf, event->val3u);
+	profiler_log_encode_string(buf, event->descr);
 }
 
 EVENT_INFO_DEFINE(data_event,
-		  ENCODE(PROFILER_ARG_S8, PROFILER_ARG_S16, PROFILER_ARG_S32,
-			 PROFILER_ARG_U8, PROFILER_ARG_U16, PROFILER_ARG_U32),
+		  ENCODE(PROFILER_ARG_S8, PROFILER_ARG_S16, PROFILER_ARG_S32, PROFILER_ARG_U8,
```

Comment:
Consider placing enum and description related to the sting in a separate line (for readability)